package.json

```diff
@@ -13,10 +14,14 @@
 		"node": ">=6"
 	},
 	"scripts": {
-		"test": "xo && ava"
+		"build": "rollup -c build/rollup.config.js",
+		"lint": "xo lib test",
+		"test": "yarn build && yarn lint && yarn test:lib && yarn test:dist",
```

Comment:
Don't compile the test files. They can just be source. Only the `index.js` should be rolled up.

Reply:
I am not compiling the test files afaik (I only lint them)